### PR TITLE
Adjust project bootstrap scripts to be windows-friendly

### PR DESCRIPTION
**Pull request by EmiM**
### Pull Request Checklist

- [x] I have linked this PR to related GitHub issue.
- [x] I have updated the CHANGELOG.md file with relevant changes (the file is located at the root of monorepo).

### CODE REVIEW

**Review by Kacper-RF — COMMENTED** [review]
Setup environment step in E2E and check workflow failed

File: packages/backend/package.json [hunk 1]
```diff
@@ -9,14 +9,18 @@
...
-		"webpack": "webpack --env mode=development && cp ./lib/bundle.cjs ../backend-bundle/bundle.cjs",
+		"webpack": "run-script-os",
+		"webpack:default": "webpack --env mode=development && cp ./lib/bundle.cjs ../backend-bundle/bundle.cjs",
 		"webpack:prod": "webpack --env mode=production && cp ./lib/bundle.cjs ../backend-bundle/bundle.cjs",
-		"applyPatches": "patch -f -p0 < ./electron-fetch.patch || true && patch -f -p0 --forward --binary < ./parse-duration.patch || true && patch -f -p0 --forward --binary < ./parse-duration-esm.patch || true",
+		"webpack:windows": "webpack --env mode=development && xcopy .\\lib\\bundle.cjs ..\\backend-bundle\\ /Y",
+		"webpack:prod:windows": "webpack --env mode=production && xcopy .\\lib\\bundle.cjs ..\\backend-bundle\\ /Y",
+		"applyPatches": "run-script-os",
+		"applyPatches:default": "patch -f -p0 < ./electron-fetch.patch || true && patch -f -p0 --forward --binary < ./parse-duration.patch || true && patch -f -p0 --forward --binary < ./parse-duration-esm.patch || true",
+		"applyPatches:windows": "git apply ./electron-fetch-git.patch --whitespace=fix --reject --verbose --no-index --ignore-space-change --inaccurate-eof || cd .",
```
Comment (leblowl) [inline packages/backend/package.json:19]: Should we just use git apply on all systems to keep things simple?

File: packages/mobile/package.json [hunk 2]
```diff
@@ -12,12 +12,18 @@
     "lint": "eslint --ext .jsx,.js,.ts,.tsx ./src/ --fix",
     "lint-ci": "eslint --ext .jsx,.js,.ts,.tsx ./src/",
     "gen": "plop",
-    "prepare-backend-assets": "mkdir -p ./nodejs-assets/nodejs-project && cp ../backend/lib/bundle.cjs ./nodejs-assets/nodejs-project/bundle.cjs",
+    "prepare-backend-assets": "run-script-os",
+    "prepare-backend-assets:default": "mkdir -p ./nodejs-assets/nodejs-project && cp ../backend/lib/bundle.cjs ./nodejs-assets/nodejs-project/bundle.cjs",
+    "prepare-backend-assets:windows": "mkdir .\\nodejs-assets\\nodejs-project || cd . && xcopy ..\\backend\\lib\\bundle.cjs .\\nodejs-assets\\nodejs-project\\ /Y",
```
Comment (leblowl) [inline packages/mobile/package.json:17]: Is this change necessary, doesn't powershell understand both `mkdir`, `cd` and `cp`?

**Review by leblowl — COMMENTED** [review]
Thanks for looking into this @EmiM!

**Comment by leblowl** [comment]
Does this fix: https://github.com/TryQuiet/quiet/issues/1818?

File: packages/backend/package.json (hunk 1 above)
Comment (EmiM) [reply packages/backend/package.json:19]: I totally agree that we should use one tool instead of two for the same purpose but **I think** patch files differ in format between those generated by patch and git.
I'd say: let's do this but not in scope of this MR but separately.

Comment (EmiM) [reply packages/backend/package.json:19]: What's more - I still have a problem with handling git's patch correctly (on Windows at least). Applying works fine on windows machine that patch was created on but fails on CI or another windows machine. I suppose git may be changing encoding? I am not sure yet.

File: packages/mobile/package.json (hunk 2 above)
Comment (EmiM) [reply packages/mobile/package.json:17]: `mkdir` differs in behaviour on Windows - it throws error and stops the process when directory already exists. That's why the `|| cd .` hack is here.
Also - yes, powershell has `cp` command but according to people's outputs in the task I saw that they were using regular cmd and cmd in windows does not have `cp`.
Of course we could add suggestion/requirement to the readme to use powershell but I decided that adjusting scripts to both cmd and powershell is not over the top since other commands (like mkdir) also work differently on Windows and Unix-like OSs.

**Comment by EmiM** [comment]
@leblowl almost but I added a note to desktop readme. There still may be an issue with patch.

**Review by vinkabuki — APPROVED** [review]

**Comment by vinkabuki** [comment]
LGTM, I've tested it on my local windows machine which used to fail - now it works, I am able to open and use the app

File: packages/backend/package.json (hunk 1 above)
Comment (leblowl) [reply packages/backend/package.json:19]: Ok sounds good. Does the issue on CI break our Windows CI in any way?

File: packages/mobile/package.json (hunk 2 above)
Comment (leblowl) [reply packages/mobile/package.json:17]: Ok, sounds good. Another alternative for the future could be having prepare scripts written in Javascript, so that we can maintain the same code for all platforms.

**Comment by leblowl** [comment]
I'd recommend squashing commits to keep the git history cleaner on develop

File: packages/backend/package.json (hunk 1 above)
Comment (EmiM) [reply packages/backend/package.json:19]: It should not. E2E tests pass, other checks too.
